YouTubeSpammerPurge.py

```diff
@@ -34,7 +34,7 @@
 ### IMPORTANT:  I OFFER NO WARRANTY OR GUARANTEE FOR THIS SCRIPT. USE AT YOUR OWN RISK.
 ###             I tested it on my own and implemented some failsafes as best as I could,
 ###             but there could always be some kind of bug. You should inspect the code yourself.
-version = "1.3.3"
+version = "1.3.4"
```

Comment:
Pretty sure Thio is the one to update versions..

Reply:
It's also reverting 1 or 2 versions 
https://github.com/ThioJoe/YT-Spammer-Purge/blob/8b93b54e786e5032412979b51247e0e70c0d86d5/YTSpammerPurge.py#L39

Reply:
> It's also reverting 1 or 2 versions

Yeah, only reverting a couple🙏